### DISCUSSION TITLE
Button to clear "new" tag

### DISCUSSION THREAD

**Discussion by Trahloc** [civitai/civitai#190] [Ideas]
I searched but I wasn't able to find anything to match how to do this, sorry if I failed to find it.

I love the "new" and "updated" tags on items.  Unfortunately I can't figure out how to clear them.  If I click on them, they remain. If I log out and back in they remain.  Once I've click on an item I expect the "new" tag to clear itself.  It'd be great if there was a button to "mark all as seen/read" just like the notifications have.

**Comment by JustMaier** [comment]
They actually aren't clearable right now. Instead they're hardcoded to show for 24 hours after the model is released or updated.

This seems like a reasonable idea though, I'll get it onto our list.
